docs/react-testing-library/setup.mdx

```diff
@@ -155,6 +155,11 @@ elements by `data-cy`, a "test ID" convention mentioned in the
 [Cypress.io](https://docs.cypress.io/guides/references/best-practices.html#Selecting-Elements)
 documentation.
 
+  <Tabs groupId="test-utils" defaultValue="js" values={[ {label: 'Javascript',
+  value: 'js'}, {label: 'Typescript', value: 'ts'}, ]}>
```

Comment:
[Nit]
Maybe we should write `Javascript` and `Typescript` with a capital `s`?
(Not sure we're following this rule in other places also, so this really isn't important)